src/shims/unix/foreign_items.rs

```diff
@@ -252,6 +252,38 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriInterpCxExt<'mir, 'tcx> {
                 this.write_scalar(result, dest)?;
             }
 
+            "reallocarray" => {
+                // TODO: Since macOs tends to grab FreeBSD's userland pieces,
+                // it might get support at some point.
```

Comment:
```suggestion
                // Currently this function does not exist on all Unixes, e.g. on macOS.
```
It's not really our concern what Apple does with macOS. We'll add it if that happens, but until then it's not a TODO on our side.